src/Catalog/NuGet.Services.Metadata.Catalog.csproj

```diff
@@ -106,17 +147,50 @@
       <HintPath>..\..\packages\VDS.Common.1.5.0\lib\net40-client\VDS.Common.dll</HintPath>
       <Private>True</Private>
     </Reference>
+    <Reference Include="xunit.assert, Version=2.2.0.3545, Culture=neutral, PublicKeyToken=8d05b1bb7a6fdb6c, processorArchitecture=MSIL">
+      <HintPath>..\..\packages\xunit.assert.2.2.0\lib\netstandard1.1\xunit.assert.dll</HintPath>
+    </Reference>
   </ItemGroup>
   <ItemGroup>
+    <Compile Include="AggregateCursor.cs" />
     <Compile Include="BatchCollector.cs" />
     <Compile Include="CatalogIndexEntry.cs" />
     <Compile Include="CatalogIndexReader.cs" />
     <Compile Include="CloudBlobStorageExtensions.cs" />
     <Compile Include="CommitCollector.cs" />
     <Compile Include="CommitMetadata.cs" />
+    <Compile Include="Helpers\AuditRecordHelpers.cs" />
+    <Compile Include="Helpers\FeedHelpers.cs" />
+    <Compile Include="LogEvents.cs" />
+    <Compile Include="Monitoring\Model\NonhijackableV2HttpMessageHandlerWrapper.cs" />
+    <Compile Include="Monitoring\NuGetLogger.cs" />
+    <Compile Include="Monitoring\Resources\IPackageTimestampMetadataResource.cs" />
+    <Compile Include="Monitoring\ValidationTestCollector.cs" />
+    <Compile Include="Monitoring\Model\PackageTimestampMetadata.cs" />
+    <Compile Include="Monitoring\Resources\PackageTimestampMetadataResourceV2.cs" />
+    <Compile Include="Monitoring\Providers\PackageTimestampMetadataResourceV2Provider.cs" />
+    <Compile Include="Monitoring\Resources\PackageTimestampMetadataResourceV3.cs" />
+    <Compile Include="Monitoring\Providers\PackageTimestampMetadataResourceV3Provider.cs" />
+    <Compile Include="Monitoring\Providers\NonhijackableV2HttpHandlerResourceProvider.cs" />
+    <Compile Include="Monitoring\Validation\Test\EndpointToTest.cs" />
+    <Compile Include="Monitoring\Validation\Test\EndpointToTestFactory.cs" />
+    <Compile Include="Monitoring\Validation\Test\PackageValidationTestRunner.cs" />
+    <Compile Include="Monitoring\Validation\Test\ResourceToTest.cs" />
+    <Compile Include="Monitoring\Validation\Test\EndpointValidationTestRunner.cs" />
+    <Compile Include="Monitoring\Validation\Result\PackageValidationTestResult.cs" />
+    <Compile Include="Monitoring\Validation\Result\EndpointValidationTestResult.cs" />
+    <Compile Include="Monitoring\Validation\Test\RegistrationValidationTests.cs" />
+    <Compile Include="Monitoring\Validation\Test\ValidationTestRunner.cs" />
+    <Compile Include="Monitoring\Validation\Result\ValidationTestResult.cs" />
+    <Compile Include="Monitoring\Validation\Test\ValidationTestRunnerFactory.cs" />
+    <Compile Include="Monitoring\Validation\Test\ValidationTestRunData.cs" />
+    <Compile Include="Monitoring\Validation\Test\ValidationTestUtility.cs" />
```

Comment:
Why the Monitoring and the test need to be in the catalog project? In my opinion I would decouple the Monitoring and definitely the testing.